test/TestOrchestrations.cs

```diff
@@ -80,6 +80,33 @@ public static string SayHelloInline([OrchestrationTrigger] DurableOrchestrationC
             return currentValue;
         }
 
+        public static async Task<int> InfiniteCounter([OrchestrationTrigger] DurableOrchestrationContext ctx)
+        {
+            int currentValue = ctx.GetInput<int>();
+
+            bool done = false;
+            while (!done)

```

Comment:
Can we model the test orchestration as something other than a potentially infinite actor? What we have here is actually a bit of an anti-pattern. In general we advise that infinite loops because of the unbounded CPU and memory overhead they create if the loop is not terminated. Obviously this works fine as a test case because we know that it will never actually be treated like an infinite orchestration, but I worry about others looking at this as a valid use case and going in a bad direction.

Reply:
Maybe we can instead model it after your document batch processing scenario?

Reply:
Ah, makes sense. Sure thing I'll make this more closely model the "file set" use case.